### PR TITLE
Update form

### PR DESCRIPTION
**Pull request by nbeteram**
The form has been updated. When you fill in 2 job applications, the first one will be replaced by the second one.

### CODE REVIEW

**Review by jay-tux — APPROVED** [review]
:+1:

File: backend/routes/form.ts [hunk 1]
```diff
@@ -1255,6 +1255,28 @@ async function addJobApplicationToDatabase(
...
+            latestJobApplication.job_application_id
+        );
+        await ormAtt.deleteAllAttachmentsForApplication(
+            latestJobApplication.job_application_id
+        );
+        await ormJoSk.deleteSkillsByJobApplicationId(
+            latestJobApplication.job_application_id
+        );
+        await ormJo.deleteJobApplication(
+            latestJobApplication.job_application_id
```
Comment (BramDevlaminck) [inline backend/routes/form.ts:1276]: for efficiency reasons you could use `await Promise.all` for all the queries that don't depend on each other.
This means you only have to await the `deleteJobApplication` extra.

small example (this is actually how it should be here):
```ts
await Promise.all([ormAppRo.deleteAppliedRolesByJobApplication(
              latestJobApplication.job_application_id
          ), ormAtt.deleteAllAttachmentsForApplication(
              latestJobApplication.job_application_id
          ), ormJoSk.deleteSkillsByJobApplicationId(
              latestJobApplication.job_application_id
          )
];
await ormJo.deleteJobApplication(
              latestJobApplication.job_application_id
);
```

**Review by BramDevlaminck — CHANGES_REQUESTED** [review]
good job, we should only use that Promise.all in that one case

**Review by BramDevlaminck — APPROVED** [review]
awesome!

**Pull request merged**